music21/prebase.py

```diff
@@ -264,26 +263,8 @@ def _reprInternal(self) -> str:
         >>> p = prebase.ProtoM21Object()
         >>> p._reprInternal()
         'object at 0x112590380'
-
-        If an object has `.id` defined and `x.id` is not the same as `id(x)`
```

Comment:
Would prefer a note here saying that "a more complex `_reprInternal` that handles the case of objects with .id defined is found in Music21Object"  Thanks!